### PR TITLE
Create are_ids_eq.mdx

### PR DESCRIPTION
**Pull request by kbinreallife**

### CODE REVIEW

File: docs/scripting/scripts.lib/are_ids_eq.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: scripts.lib().are_ids_eq()
+---
+
+### Description
+
+Compares two strings and evaluates if they are equal. Useful for comparing MongoDB Object IDs.
```
Comment (tukib) [inline docs/scripting/scripts.lib/are_ids_eq.mdx:7]:
```suggestion
Compares two strings or ObjectIDs and evaluates if they are equal. Useful for comparing MongoDB documents by IDs.
```

File: docs/scripting/scripts.lib/are_ids_eq.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+```js
+return #fs.scripts.lib().are_ids_eq("65c982ee9edd1f997c9806ac", "65c982ee9edd1f997c9806a0");
+```
+
+### Parameters
+
+#### id1
+
+The first string value.
```
Comment (tukib) [inline docs/scripting/scripts.lib/are_ids_eq.mdx:19]:
```suggestion
The first string or ObjectID value.
```

File: docs/scripting/scripts.lib/are_ids_eq.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+#### id2
+
+The second string value.
```
Comment (tukib) [inline docs/scripting/scripts.lib/are_ids_eq.mdx:23]:
```suggestion
The second string or ObjectID value.
```

File: docs/scripting/scripts.lib/are_ids_eq.mdx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+## Example
+
+```js
+function(context, args) {
+  let l = #fs.scripts.lib();
+  let id1 = "65c982ee9edd1f997c9806ac";
+  let id2 = "65c982ee9edd1f997c9806a0";
+  return l.are_ids_eq(id1, id2);
+}
+```
```
Comment (tukib) [inline docs/scripting/scripts.lib/are_ids_eq.mdx:42]:
````suggestion
```

```js
function(context, args) {
  let l = #fs.scripts.lib();
  let id1 = { $oid: "65c982ee9edd1f997c9806ac" };
  let id2 = { $oid: "65c982ee9edd1f997c9806a0" };
  return l.are_ids_eq(id1, id2);
}
```
````

**Review by tukib — DISMISSED** [review]
I think we should also document the behaviour with ObjectIDs too.

**Review by ellipsisasterisk — APPROVED** [review]

**Pull request merged**